src/config_parser.ml

```diff
@@ -469,9 +473,11 @@ let pp_dhcp_opt ppf = function
       Fmt.pf ppf "option:router,%a"
         Fmt.(list ~sep:(any ",") Ipaddr.V4.pp)
         servers
-  | _ ->
-      (* TODO should print <ID>,<VAL> - but there's no dhcp_option_to_option_code in dhcp_wire *)
-      assert false
+  | Dhcp_wire.Dns_servers servers ->
+      Fmt.pf ppf "option:dns-server,%a"
+        Fmt.(list ~sep:(any ",") Ipaddr.V4.pp)
+        servers
+  | x -> Fmt.pf ppf "option:%s" (Dhcp_wire.dhcp_option_to_string x)
```

Comment:
This is not quite correct. For example, `dhcp-option=option:irc-server,127.0.0.1` is parsed and then pretty printed as `dhcp-option=option:IRC servers 127.0.0.1`. But as long we don't parse the unhandled cases it should not be a problem in our application.

Reply:
yes, this is a stop-gap solution until we have something better. the assert false was frightening since I hit it once or twice in tests...